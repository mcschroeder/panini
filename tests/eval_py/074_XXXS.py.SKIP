def f074(s: str):
  i = 1
  t = ""
  while i < len(s):
    t += s[i]
    i += 1
  assert len(t) == 2

# skipped, because this example is out of scope:
# we are trying to constrain the input string s by asserting an
# additional semantic restriction on the output of the parsing of s
# (which happens to be another string t, but that's irrelevant)
