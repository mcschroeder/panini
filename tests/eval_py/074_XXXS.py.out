f074 : {s:𝕊 | s ∈ ...*} → 𝟙